src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferenceCache.cs

```diff
@@ -96,20 +96,9 @@ public static SymbolInfo GetSymbolInfo(SemanticModel model, SyntaxNode node, Can
             ISyntaxFactsService syntaxFacts, Document document, VersionStamp version, SyntaxNode root,
             SourceText content, string text, Func<SyntaxToken, bool> candidate, CancellationToken cancellationToken)
         {
-            if (text.Length > 0)
-            {
-                using (var positions = SharedPools.BigDefault<List<int>>().GetPooledObject())
-                {
-                    if (SyntaxTreeIdentifierInfo.TryGetIdentifierLocations(document, version, text, positions.Object, cancellationToken))
-                    {
-                        return GetTokensFromText(root, positions.Object, text, candidate, cancellationToken);
-                    }
-                }
-
-                return GetTokensFromText(syntaxFacts, root, content, text, candidate, cancellationToken);
-            }
-
-            return ImmutableArray<SyntaxToken>.Empty;
+            return text.Length > 0
+                ? GetTokensFromText(syntaxFacts, root, content, text, candidate, cancellationToken)
+                : ImmutableArray<SyntaxToken>.Empty;
```

Comment:
So - this means we don't use the bloom filters at all for FAR anymore?  Should we be fixing that instead of removing this code?